### PR TITLE
added a column 'ProductCategory' in ProductTable

### PR DESCRIPTION
**Pull request by MuhammadZain795**
ProductCategory will contain type of product when a new product is added. Specific words can be decided for categories of products like Art,Books,Clothing etc.

### CODE REVIEW

File: Src/DBProj/Tables/Products.sql [hunk 1]
```diff
@@ -8,6 +8,7 @@
     [ModifiedOn]  DATETIME     NULL,
     [ModifiedBy]  INT          NULL,
     [IsActive]    BIT          NOT NULL,
+    [ProductCategory] VARCHAR(100) NOT NULL,
```
Comment (bilalshahzad139) [inline Src/DBProj/Tables/Products.sql]: @MuhammadZain795 
Please Create a separate table dbo.ProductCategory and use its ProductCategoryID as reference key in Product table.

Comment (MuhammadZain795) [reply Src/DBProj/Tables/Products.sql]: ok Sir! @bilalshahzad139

File: Src/DBProj/DBProj.sqlproj [hunk 2]
```diff
@@ -68,11 +68,15 @@
     <Build Include="Tables\Products.sql" />
     <Build Include="Tables\Users.sql" />
     <Build Include="Procedures\dbo_SaveUser.sql" />
+    <Build Include="Tables\ProductCategory.sql" />
```
Comment (bilalshahzad139) [inline Src/DBProj/DBProj.sqlproj:71]: ProductCategory.sql file is missing in commit, it seems it was not marked added while commiting through tortoisegit

Comment (MuhammadZain795) [reply Src/DBProj/DBProj.sqlproj:71]: ok I do it again.

**Comment by MuhammadZain795** [comment]
Sir! is it ok now? @bilalshahzad139

**Pull request merged**